tool/planet/constants.go

```diff
@@ -358,6 +428,96 @@ var K8sSearchDomains = []string{
 	"kube-system.svc.cluster.local",
 }
 
+// KubeletConfig specifies the default kubelet configuration
+var KubeletConfig = kubeletconfig.KubeletConfiguration{
+	TypeMeta:               KubeletTypeMeta,
+	Address:                "0.0.0.0",
+	Port:                   10250,
+	MakeIPTablesUtilChains: utils.BoolPtr(true),
+	HealthzBindAddress:     "0.0.0.0",
+	HealthzPort:            utils.Int32Ptr(10248),
+	ClusterDomain:          "cluster.local",
+	EventRecordQPS:         utils.Int32Ptr(0),
+	FailSwapOn:             utils.BoolPtr(false),
+	ReadOnlyPort:           0,
+	TLSCertFile:            "/var/state/apiserver.cert",
```

Comment:
We have a bunch of "/var/state/*" constants in tool/planet/constants.go, maybe add these too?